cmd/kubelet/app/options/options.go

```diff
@@ -59,7 +59,7 @@ func NewKubeletServer() *KubeletServer {
 
 // AddFlags adds flags for a specific KubeletServer to the specified FlagSet
 func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
-	fs.StringVar(&s.Config, "config", s.Config, "Path to the config file or directory of files")
+	fs.StringVar(&s.PodManifestPath, "config", s.PodManifestPath, "Path to the config file or directory of files")
```

Comment:
Not necessary.

On Wed, Aug 3, 2016 at 3:38 PM, Michael Taufen <notifications@github.com>
wrote:

> In cmd/kubelet/app/options/options.go
> <https://github.com/kubernetes/kubernetes/pull/29999#discussion_r73434551>
> :
>
> > @@ -59,7 +59,7 @@ func NewKubeletServer() *KubeletServer {
> >
> >  // AddFlags adds flags for a specific KubeletServer to the specified FlagSet
> >  func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
> > -	fs.StringVar(&s.Config, "config", s.Config, "Path to the config file or directory of files")
> > +	fs.StringVar(&s.PodManifestPath, "config", s.PodManifestPath, "Path to the config file or directory of files")
>
> Sure. Would we need to loop someone else in for an API review on that?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/29999/files/f2c4ce0caeb751c77fc2ac0480d67e0f34a7955b#r73434551>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGvIKOiy2-KAD7f2AU8OvxH9ZH8xM10iks5qcRiAgaJpZM4Jb2IN>
> .
>